challenges/functions.js

```diff
@@ -7,25 +7,36 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
-
+function consume (x, y, cb) {
+  console.log(cb(x,y));
```

Comment:
Replace console.log with `return` to get rid of side effects that will try to console.log the `undefined` each time consume is called.